pkg/maven/maven_test.go

```diff
@@ -110,5 +111,23 @@ func TestDownloadSettingsFromURL(t *testing.T) {
 		downloadSettingsFromURL("anyURL", "settings.xml", &mockClient)
 		assert.True(t, hasFailed, "expected command to exit with fatal")
 	})
+}
+
+func TestGetTestModulesExcludes(t *testing.T) {
+	t.Run("Should return excludes for unit- and integration-tests", func(t *testing.T) {
+		os.Mkdir("unit-tests", 0777)
+		defer os.Remove("unit-tests")
+		os.Create("unit-tests/pom.xml")
+		defer os.Remove("unit-tests/pom.xml")
+
+		os.Mkdir("integration-tests", 0777)
+		defer os.Remove("integration-tests")
+		os.Create("integration-tests/pom.xml")
+		defer os.Remove("integration-tests/pom.xml")
```

Comment:
This doesn't look so good to me. I would either create test data in the repo (there are examples), or create a temp folder (there is an os-method for that I think) and Chdir() into the folder for the test. The benefit of having test-data in the repo would be that the test runs slightly faster. Also, we could extend one of the existing test data folders so it contains stuff usable for more integration tests. A third alternative would be to use mocking. I believe there are examples for this as well.

Reply:
agree. I added test data to resources, with that we can simulate a maven project and fill the poms as we need.